<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>heapsort_add</class>
 <widget class="QDialog" name="heapsort_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>251</width>
     <height>222</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="editAuthor"/>
      </item>
      <item>
       <widget class="QLineEdit" name="editName"/>
      </item>
      <item>
       <widget class="QLineEdit" name="editGenre"/>
      </item>
      <item>
       <widget class="QLineEdit" name="editPublishingHouse"/>
      </item>
      <item>
       <widget class="QLineEdit" name="editPublicatioYear"/>
      </item>
      <item>
       <widget class="QLineEdit" name="editNumberOfPages"/>
      </item>
      <item>
       <widget class="QLineEdit" name="editNumberOfCopies"/>
      </item>
      <item>
       <widget class="QLineEdit" name="editNumberOfCopiesForReaders"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelAuthor">
        <property name="text">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelGenre">
        <property name="text">
         <string>Genre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPublishingHouse">
        <property name="text">
         <string>PublishingHouse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPublicatioYear">
        <property name="text">
         <string>PublicatioYear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelNumberOfPages">
        <property name="text">
         <string>NumberOfPages</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelNumberOfCopies">
        <property name="text">
         <string>NumberOfCopies</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelNumberOfCopiesForReaders">
        <property name="text">
         <string>NumberOfCopiesForReaders</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>166</width>
     <height>21</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
